amp-story/templates/ampmars/json/bookend.json

```diff
@@ -0,0 +1,40 @@
+{
+    "bookendVersion": "v1.0",
+    "shareProviders": ["facebook", "twitter", "email"],
```

Comment:
Facebook requires an application ID, so it would be nice to provide more realistic JSON, faking out only the ID itself.   
 Additionally, it would be great to utilize more advanced features where possible, like the ability to specify provider-specific options.  So, can you change this to:

```javascript
  "shareProviders": [
    {
      "provider": "twitter",
      "text": "The Mars Cruise -- Get ready for the trip of your life!"
    },
    {
      "provider": "facebook",
      "app_id": "12345678910"
    },
    "email"
  ],

  // ...
```

note: feel free to change the text of the Twitter share provider to something that feels like would be good copy if you were to click the tweet sharing option to share this fake publisher's story 😄